server/datastore/mysql/migrations/tables/20221115104546_AddCronStatsTable.go

```diff
@@ -21,6 +21,7 @@ func Up_20221115104546(tx *sql.Tx) error {
 			created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
 			updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
 			INDEX idx_cron_stats_name_created_at (name, created_at))
```

Comment:
Let's pull the closing ) onto a new line. I realize that makes the diff larger but otherwise this looks bugged on first glance.